static/templates/admin/resultTableView.html

```diff
@@ -5,17 +5,20 @@
 			<table class="result_table">
 				<colgroup>
 					<% _.each(resultFields, function(resultField) { %>
-						<col class="<%= resultField.colClass %>">
-					<% }); %>
+						<% if (resultField.colClass == "collection_id" && container.type != "AdminUnit") { %>
```

Comment:
generally we avoid having empty conditionals. We could invert this so we don't need the else.

`if (!(resultField.colClass == "collection_id" && container.type != "AdminUnit"))`
 or 
`if (resultField.colClass != "collection_id" || container.type == "AdminUnit")`